net/usrsock/usrsock_sendto.c

```diff
@@ -172,6 +172,7 @@ static int do_sendto_request(FAR struct usrsock_conn_s *conn,
 
   req.head.reqid = USRSOCK_REQUEST_SENDTO;
   req.usockid = conn->usockid;
+  req.flags = flags;
```

Comment:
Please submit the related patch to:
apps/examples/usrsocktest/
apps/wireless/gs2200m/
apps/netutils/usrsock_rpmsg/

Reply:
Okay, I'll push them later.